bmgui/parse_corpus_page.py

```diff
@@ -432,7 +439,10 @@ def _deduplicate_corpus_parsing():
                                                      inst_filter_list = institutions_filter_list)
         save_parsing_dict(concat_parsing_dict, concat_parsing_path,
                           item_filename_dict, parsing_save_extent)
-        dedup_parsing_dict  = bp.deduplicate_parsing(concat_parsing_dict)
+        dedup_parsing_dict  = bp.deduplicate_parsing(concat_parsing_dict, 
+                                                     norm_inst_status = False,
+                                                     inst_types_file_path = inst_types_file_path,
+                                                     country_affiliations_file_path = institute_affil_file_path)
```

Comment:
Those parameters are not recognized as functions arguments for the deduplicate_parsing() method.

Reply:
This is related to the BiblioParsing new version as already mentionned.